Tasks/AzureRmWebAppDeploymentV4/deploymentProvider/BuiltInLinuxWebAppDeploymentProvider.ts

```diff
@@ -94,8 +99,17 @@ export class BuiltInLinuxWebAppDeploymentProvider extends AzureRmWebAppDeploymen
 
     public async UpdateDeploymentStatus(isDeploymentSuccess: boolean) {
         if(this.kuduServiceUtility) {
-            await super.UpdateDeploymentStatus(isDeploymentSuccess);
-            if(this.zipDeploymentID && this.activeDeploymentID && isDeploymentSuccess) {
+           
+ 	        this.activeDeploymentID =  this.kuduServiceUtility.getDeploymentID();
+            if(isDeploymentSuccess == false){
```

Comment:
@ozanovus  here **azurermwebappdeployment.ts** the **isDeploymentSuccess** is set to true in the declaration and we should not check the condition is false. So I am checking if the **isDeploymentSuccess** is false am calling this **await super.UpdateDeploymentStatus(isDeploymentSuccess)**.

Reply:
@v-nagarajku , actually I'm confused a little bit. Why did we give up the call "`await super.UpdateDeploymentStatus(isDeploymentSuccess);`"? Instead of calling this one, It seems like you have copied same code piece from `AzureRmWebAppDeploymentProvider.ts`.

Reply:
@ozanovus  we have to stop calling **super.UpdateDeploymentStatus(isDeploymentSuccess);**  as this function will call an API which is not required when the deployment was success. This was the main agenda for this PR.

Reply:
As we already have discussed in the meeting, when you push your commit related to moving logic to parent method, we can mark as solved. Thanks for your effort!